io.elementary.mail.yml

```
@@ -108,7 +110,13 @@ modules:
     sources:
       - type: git
         url: https://gitlab.gnome.org/GNOME/libhandy.git
-        tag: '1.0.1'
+        tag: '1.2.0'
+
+  - name: granite
+    buildsystem: meson
+    sources:
+     - type: git
+       url: https://github.com/elementary/granite.git
 
```

We can probably just target 6.0 now:
```suggestion
       url: https://github.com/elementary/granite.git
       tag: '6.0.0'
```